### ISSUE TITLE
new start

### ISSUE THREAD

**Issue by munawarbhutto34** [munawarbhutto34/Teachers_Availability#3]
sir we have change the project so other member saad mansoor will create new repo and add all of us including you in that repo so we will doing our work on it.

**Comment by khanshakeeb** [comment]
@munawarbhutto34 can you share idea ? What you are going to build

**Comment by munawarbhutto34** [comment]
Yes sir finally we are building a sinple menu but with different functions
like calculater converter etc

On Dec 6, 2017 8:13 PM, "Muhammad Shakeeb Khan" <notifications@github.com>
wrote:

@munawarbhutto34 <https://github.com/munawarbhutto34> can you share idea ?
What you are going to build

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/munawarbhutto34/Teachers_Availability/issues/3#issuecomment-349669120>,
or mute the thread
